cli/cmd/remove.go

```diff
@@ -0,0 +1,55 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/configure"
+	"github.com/tarantool/tt/cli/modules"
+	"github.com/tarantool/tt/cli/remove"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// NewRemoveCmd creates remove command.
+func NewRemoveCmd() *cobra.Command {
+	var removeCmd = &cobra.Command{
+		Use:   "remove what",
```

Comment:
Be consistent with other commands:
remove <PROGRAM>

Reply:
changed